### PR TITLE
[REVIEW] add getSizeInBytes for DType

### PR DESCRIPTION
**Pull request by wbo4958**
For some cases, we need to get column info and pass it to JNI layer to do some cuda operations.

### CODE REVIEW

**Comment by GPUtester** [comment]
Please update the changelog in order to start CI tests.

View the gpuCI docs [here](https://docs.rapids.ai/gpuci#open-pr-or-update-pr).

**Comment by nartal1** [comment]
@wbo4958 , Please update changelog

**Comment by wbo4958** [comment]
> @wbo4958 , Please update changelog

Thx, Done.

File: java/src/main/java/ai/rapids/cudf/DType.java [hunk 1]
```diff
@@ -78,6 +78,10 @@ public int getNativeId() {
     return nativeId;
   }
 
+  public int getSizeInBytes() {
```
Comment (revans2) [inline java/src/main/java/ai/rapids/cudf/DType.java:81]: Can we add some docs to this?  Especially explaining that this only works for fixed width types.  Variable width types like strings the value is undefined and should be ignored.

Comment (wbo4958) [reply java/src/main/java/ai/rapids/cudf/DType.java:86]: thx ,Done

**Pull request merged**